src/frequenz/client/dispatch/_internal_types.py

```diff
@@ -25,6 +25,8 @@
     component_selector_to_protobuf,
 )
 
+# pylint: enable=no-name-in-module
+
```

Comment:
This doesn't look right here, but you could also just disable it globally in the whole project as `mypy` will check for this anyway.

Reply:
I think this is mostly at this place due to `black`

Reply:
Yeah, it is weird, it has to be just before the next line.